### DISCUSSION TITLE
Roles & Responsibilities

### DISCUSSION THREAD

**Discussion by ZaradarBH** [classic-terra/core#117] [Team Posts]
**L1 Architects** 
@edk208, @inon-man @ZaradarBH

**L1 Lead Developers** 
@inon-man

**L1 Senior Developers** 
@edk208, @inon-man @nghuyenthevinh2000 @ZaradarBH @fragwuerdig

**L1 Junior Developers** 
@Ragepie @shackra

**Project Managers**
@LuncBurner @edk208 @ZaradarBH

**Comment by dellacostamoveis-code** [comment]
Criar um projeto de recompensa para quem ficar por tempos determinados sem sacar valores.
Tipo escalonado
Ou seja ser diferente,ser inovador.
